### PR TITLE
blockchain gambling md file updated

### PR DESCRIPTION
**Pull request by haiweiosu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+[![Stories in Ready](https://badge.waffle.io/haiweiosu/cs5356.png?label=ready&title=Ready)](https://waffle.io/haiweiosu/cs5356)
```
Comment (amfleming) [inline README.md:1]: Surely you are not suggesting we put your badge on README.md for the class?

**Comment by amfleming** [comment]
Hi Haiwei - a couple problems with this PR
- You are modifying README.md when I don't think you want to
- There are 5 commits when I think you only want to PR 1

And looking at your Trello (cool) it is not clear what you are going to demo on weds / what your first story is 

(I see the _user stories_, but what is _your_ first story - what working software will you demo on Weds?)

File: README.md (hunk 1 above)
Comment (haiweiosu) [inline README.md:1]: Sorry I made an error on that. Could you just cancel that pull request
please?

Thanks,
Haiwei

On Monday, September 21, 2015, Adam Fleming notifications@github.com
wrote:

> In README.md
> https://github.com/Cornell-CS5356-Fall2015/cs5356/pull/214#discussion_r40031057
> :
> 
> > @@ -1,3 +1,4 @@
> > +[Stories in Ready](https://waffle.io/haiweiosu/cs5356)
> 
> Surely you are not suggesting we put your badge on README.md for the class?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/Cornell-CS5356-Fall2015/cs5356/pull/214/files#r40031057
> .

## 

Sent from Gmail Mobile

Comment (dblock) [inline README.md:1]: Hit Close to any PRs you don't want merged, thx!

**Comment by haiweiosu** [comment]
Just closed PRs.

File: README.md (hunk 1 above)
Comment (haiweiosu) [inline README.md:1]: Just closed PRs. Sorry about the confuse.

**Pull request closed without merging**